packages/ember-intl/addon/helpers/-format-base.js

```diff
@@ -46,10 +45,14 @@ function helperFactory(formatType, helperOptions = {}) {
       }
 
       const intl = get(this, 'intl');
-      let locale = get(intl, '_locale');
+      let locale = hash.locale;
+
+      if (!locale) {
+        locale = get(intl, '_locale');
 
-      if (Array.isArray(locale)) {
-        locale = locale[0];
+        if (Array.isArray(locale)) {
```

Comment:
This is another bug, I'm going to fix this as well.  It should accept an array.

https://github.com/yahoo/intl-messageformat#intlmessageformat-constructor